nautobot/dcim/signals.py

```diff
@@ -290,11 +291,11 @@ def prevent_adding_tagged_vlans_with_incorrect_mode_or_site(sender, instance, ac
 
 
 #
-# Interface tagged VLAMs
+# VirtualDeviceContext Interfaces
 #
 
 
-@receiver(m2m_changed, sender=Interface.virtual_device_contexts.through)
+@receiver(m2m_changed, sender=InterfaceVDCAssignment)
```

Comment:
Given that we have a concrete through table now, does this actually even need to be a signal? Could it be implemented as `InterfaceVDCAssignment.save()` instead?

Reply:
With this approach, we can bulk validate